docs/desktop-wpf/migration/convert-project-from-net-framework.md

```diff
@@ -8,92 +8,99 @@ ms.author: mikerou
+This article references the [Bean Trader sample app](https://github.com/dotnet/windows-desktop/tree/master/Samples/BeanTrader) because it uses a variety of dependencies similar to those real-world WPF apps might have. The app isn't large but is meant to be a step up from 'Hello World' in terms of complexity. The app will demonstrate some issues users may encounter while porting real applications. The app communicates with a WCF service so, in order for it to run properly, so you'll also need to run the BeanTraderServer project (available in the same GitHub repository) and make sure the BeanTraderClient's configuration points at the correct endpoint (by default, the sample assumes the server is running on the same machine at `http://localhost:8090`, which will be true if you just launch **BeanTraderServer** locally).
 
-Keep in mind that this sample app is meant to demonstrate .NET Core porting challenges and solutions. It's not meant to demonstrate WPF best practices. In fact, it deliberately includes some anti-patterns to make sure we come across at least a couple interesting challenges while porting.
+Keep in mind that this sample app is meant to demonstrate .NET Core porting challenges and solutions. It's not meant to demonstrate WPF best practices. In fact, it deliberately includes some anti-patterns to make sure you come across at least a couple interesting challenges while porting.
 
 ## Getting ready
 
 The primary challenge with migrating a .NET Framework app to .NET Core is that its dependencies may work differently, or not at all. Migration is much easier than it used to be; many NuGet packages now target .NET Standard. Starting with .NET Core 2.0, the .NET Framework and .NET Core surface areas have become similar. Even so, some differences (both in support from NuGet packages and in available .NET APIs) remain. The first step in migrating is to review the app's dependencies and make sure references are in a format that's easily migrated to .NET Core.
 
 ### Upgrade to `<PackageReference>` NuGet references
 
-Older .NET Framework projects typically list their NuGet dependencies in a packages.config file. The new SDK-style project file format references NuGet packages differently, though. It uses [`<PackageReference>`][package-reference] elements in the csproj file itself (rather than in a separate config file) to reference NuGet dependencies.
+Older .NET Framework projects typically list their NuGet dependencies in a packages.config file. The new SDK-style project file format references NuGet packages differently, though. It uses [`<PackageReference>`](/nuget/consume-packages/package-references-in-project-files) elements in the csproj file itself (rather than in a separate config file) to reference NuGet dependencies.
```

Comment:
```suggestion
Older .NET Framework projects typically list their NuGet dependencies in a *packages.config* file. The new SDK-style project file format references NuGet packages differently, though. It uses [`<PackageReference>`](/nuget/consume-packages/package-references-in-project-files) elements in the *csproj* file itself (rather than in a separate config file) to reference NuGet dependencies.
```